File: backend_medicines/ElectronicPrescription/Models/Presentation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ElectronicPrescription.Models
{
    public class Presentation
    {
        public int PresentationId { get; set; }

        public string Form { get; set; }

        public float Concentration { get; set; }

        public int Quantity { get; set; }

        public int? DrugId { get; set; }

        public Drug Drug { get; set; }

        public ICollection<PackageLeaflet> PackageLeaflet { get; set; }
    }
}

File: frontend-electronic-prescriptions/src/app/model/presentation.ts
import { Comment } from './comment'
import { Posology } from './posology'

export class Presentation {
    drug: string;
    medicines: string[];
    posologies: Posology[];
    form: string;
    concentration: string;
    quantity: string;
    comments: Comment[];

    constructor(
        drug: string,
        medicines: string[],
        posologies: Posology[],
        form: string,
        concentration: string,
        quantity: string,
        comments: Comment[]
    ) {
        this.drug = drug;
        this.medicines = medicines;
        this.posologies = posologies;
        this.form = form;
        this.concentration = concentration;
        this.quantity = quantity;
        this.comments = comments;
    }
}